docs/config.rst

```diff
@@ -138,9 +182,9 @@ can control. The following configuration options exist for a mapping:
 - ``prefix`` A common namespace prefix that all documents of this mapping
   share. This prefix should never conflict with prefixes of other defined
   mappings otherwise some of your documents cannot be found by Doctrine. This
-  option defaults to the bundle namespace + ``Document``, for example for an
-  application bundle called ``AcmeHelloBundle``, the prefix would be
-  ``Acme\HelloBundle\Document``.
+  option defaults to the application namespace + ``Document``, for example
+  for an application called ``App``, the prefix would be
+  ``App\Document``.
 
 - ``alias`` Doctrine offers a way to alias document namespaces to simpler,
```

Comment:
I still think we've missed deprecating this one :)

Reply:
Let's deprecate it for 5.1 and remove it from the doc.